aws-eks-cost-monitoring.md

```diff
@@ -33,6 +33,18 @@ oci://public.ecr.aws/kubecost/cost-analyzer --version <$VERSION> \
 -f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/develop/cost-analyzer/values-eks-cost-monitoring.yaml
 ```
 
+To install Kubecost on Amazon EKS cluster on AWS Graviton2 (ARM-based processor), you can run following command:
+
+```bash
+helm upgrade -i kubecost \
+oci://public.ecr.aws/kubecost/cost-analyzer --version 1.96.0 \
```

Comment:
Should we be consistent with command above and use $VERSION? I'm good either way, but seems like being consistent here.

Reply:
It was my typo when copy and paste from my testing note. I will update to $VERSION before merging tomorrow. Thank you for your review.